linux/tools.Dockerfile

```diff
@@ -10,9 +9,9 @@ ARG IMAGE_LOCATION=cdpxb787066ec88f4e20ae65e42a858c42ca00.azurecr.io/official/az
 # Copy from base build
 FROM ${IMAGE_LOCATION}
 
-ENV NODE_OPTIONS=--tls-cipher-list='ECDHE-RSA-AES128-GCM-SHA256:!RC4'
-
+RUN tdnf clean all
 RUN tdnf repolist --refresh
```

Comment:
Do we need to remove tdnf repolist refresh? Since we do a clean all? Would that refresh the repolist?

Reply:
In theory, a `tdnf clean all` and a `tdnf update` should get us the latest updates. However, it is still recommended by the Mariner team to leave it in.

Reply:
Ok sounds good